beta/iam/setup-sdk.md

```diff
@@ -39,6 +39,12 @@ implementation('com.adobe.marketing.mobile:messaging:1.2.0-beta-1-SNAPSHOT')
 implementation('com.adobe.marketing.mobile:optimize:1.0.0-SNAPSHOT')
 ```
 
+If you will be using the Messaging IAM beta extension alongside the Campaign Standard extension then the latest Campaign Standard extension must be used to resolve a compatibility issue:
```

Comment:
"If you use the Messaging IAM beta extension alongside the Campaign Standard extension, the latest Campaign Standard extension must be used..."